packages/accounts-base/accounts_client_tests.js

```
@@ -147,6 +147,126 @@ Tinytest.addAsync(
   }
 );
 
+Tinytest.addAsync(
+  'accounts - onLogin non-async callback works correctly',
+  (test, done) => {
+    const onLogin = Accounts.onLogin((loginDetails) => {
```

ESLint

[error] Replace `(loginDetails)` with `loginDetails`

(prettier/prettier)